GSA_Adapter/Convert/ToGsa/Analysis/AnalysisStage.cs

```diff
@@ -0,0 +1,92 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.Engine.Adapters.GSA;
+using BH.oM.Adapters.GSA.Analysis;
+using BH.oM.Base;
+using BH.oM.Structure.Constraints;
+using BH.oM.Structure.SectionProperties;
+using BH.oM.Structure.SurfaceProperties;
+using System;
+using System.Collections.Generic;
+
+namespace BH.Adapter.GSA
+{
+    public static partial class Convert
+    {
+        /***************************************************/
+        /**** Public  Methods                           ****/
+        /***************************************************/
+
+        private static string ToGsaString(this AnalysisStage stage, string index)
+        {
+            string command = "ANAL_STAGE";
+            string name = stage.Name;
+
+            List<Object> elements = stage.Elements;
+            string objectIds = "";
+
+            for (int i = 0; i < elements.Count; i++)
+            {
+                string id = "";
+
+                // String type
+                if (int.TryParse(elements[i].ToString(), out int idInt))
```

Comment:
This syntax (putting it as 'out int idInt' does not compile with all the versions of c# we support. You will need to define the int on a row above
```suggestion
                int idInt;
                if (int.TryParse(elements[i].ToString(), out idInt))
```